docker/coturn/Makefile

```diff
@@ -24,7 +24,7 @@ dockerify = $(strip $(if $(call eq,$(1),linux/arm32v6),linux/arm/v6,\
 # Project parameters #
 ######################
 
-COTURN_VER ?= 4.6.2
+COTURN_VER ?= 4.6.3
```

Comment:
For Docker image there is no point to change the version, since it's versioned a little bit different from the sources. It will be changed in a separate commit preparing the image release.

Reply:
Sure - just looks better when searching up the code and then seeing old version number is weird